command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
On Fri, Jul 29, 2016 at 03:00:25PM -0700, Mrunal Patel wrote:
> > @@ -0,0 +1,189 @@
> > +# Operations
> > +
> > +A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
> 
> I am not sure what the compliance tests for CLI and runtime will do
> independently if they are separate. They are tied together.

They are loosely linked.  The same container configuration
(config.json) could work with both the start-only and the create/start
split versions of the command line API (assuming we keep hooks).  The
same container configuration could work regardless of whether the
split ‘create’ is early-exit or long-running.  The same command line
API will work if/when #267 lands.  Or some breaking configuration
change lands.  You could have alternative (e.g. HTTP) APIs and have
the compliance-testing framework drive the runtime through either API.

So separate repositories sounds fine to me.  And the certification
team just says:

  We certify compliance with runtime-spec v1.0 through one of the
  following APIs:

  * command-line-runtime-api v1.0
  * …

  That certification also certifies compliance with the tested API(s).